packages/documentation-site/patternfly-docs/content/design-guidelines/charts/about/about.md

```diff
@@ -65,3 +65,15 @@ On click
 
 1. **Legend:** When a user clicks on a legend label, it becomes disabled and the color swatch is replaced with an eye-slashed icon.
 2. **Chart:** Data corresponding to the clicked legend label is hidden from view.
+
+## Develop with charts 
+
+Default styles in the [@patternfly/react-charts package](https://www.npmjs.com/package/@patternfly/react-charts) are aligned with our light theme. To use chart styles, you must also have the [@patternfly/patternfly package](https://www.npmjs.com/package/@patternfly/patternfly) installed.
+
+To utilize default chart styles, you don't need to import anything else. But, to support dark-themed charts, you must also import the stylesheet that contains dark theme styles. To do so, add this line before importing your main application component:
+
+`import '@patternfly/patternfly/patternfly-charts.css';`
+
+Once you import this file, you'll have access to [all chart variables](https://www.npmjs.com/package/@patternfly/patternfly?activeTab=code). Beyond dark theme, you could use these variables to match the style of other UI elements to your chart styles. 
```

Comment:
IMO it isn't really intuitive that "all chart variables" links to `@patternfly/patternfly` on NPM. I would probably be confused clicking on that, expecting it to take me to a list of chart variables or something like that. I might pair this sentence with the one below about filtering the all tokens table, but it's worth noting that chart variables and chart tokens aren't exactly the same.

Reply:
Okay this is great feedback! I was thinking that the patternfly-charts.css file over on NPM _was_ the chart variables, but I'm definitely treading water with variables, tokens, and friends lol

Reply:
I think my remaining question is, where is a list of our chart variables?